boards/arm/mps2_an521/board.cmake

```
@@ -12,6 +12,14 @@ set(QEMU_FLAGS_${ARCH}
   )
 board_set_debugger_ifnset(qemu)
 
+set(QEMU_FLAGS_HOST_SERIAL
```

Any reason you cannot use:
```suggestion
list(APPEND QEMU_FLAGS_EXTRA
```

here ?

Probably this line should be updated in such case to also use `list(APPEND ...)`
https://github.com/zephyrproject-rtos/zephyr/blob/f882d43b56e909465dd2c2dc904512f1e2c5cef7/boards/arm/mps2_an521/board.cmake#L24